e2e/suites/actions-available/special-permissions/test-data-permissions.ts

```diff
@@ -29,9 +29,9 @@ export const random = Utils.random();
 
 // ----- files -----
 
-const consumerContextMenu = ['Share', 'Download', 'View', 'Favorite', 'Copy', 'Manage Versions'];
-const consumerFavContextMenu = ['Share', 'Download', 'View', 'Remove Favorite', 'Copy', 'Manage Versions'];
-const consumerSharedContextMenu = ['Shared Link Settings', 'Download', 'View', 'Favorite', 'Copy', 'Manage Versions'];
+const consumerContextMenu = ['Share', 'Download', 'View', 'Favorite', 'Copy', 'Edit Aspects', 'Manage Versions'];
```

Comment:
Consumer and contributor should not see Edit Aspects.
Please revert this and check the docs.